src/main/java/net/atos/zac/app/admin/model/RESTFormulierDefinitie.java

```diff
@@ -8,16 +8,14 @@
 import net.atos.zac.zaaksturing.model.FormulierDefinitie;
 
 public class RESTFormulierDefinitie {
```

Comment:
Deze class is toch niet meer nodig? Overal waar je "RESTFormulierDefinitie.id" gebruikt gebruik je toch gewoon de FormulierDefinitie enum?